tests/UpgradeRule/PHP/fixtures/add-namespace-use-statement.testfixture

```diff
@@ -0,0 +1,52 @@
+{
+    "add-namespace": [
+        {
+            "namespace": "Upgrader\\NewNamespace",
+            "path": "/dir"
+        }
+    ]
+}
+------
+<?php
+
+interface RenamedInterface extends A, B {
+
+    public function foo() : bool;
```

Comment:
We may want to test this on a class too and also test case insensitivity (for example `Array` instead of `array`).

Reply:
Lol ... never occurred to me that words like `array`, `float` or `parent`, might be case insensitive in PHP.

Should be fixed now.